applications/matter_bridge/src/app_task.cpp

```diff
@@ -244,3 +127,44 @@ CHIP_ERROR AppTask::RestoreBridgedDevices()
 	}
 	return CHIP_NO_ERROR;
 }
+
+CHIP_ERROR AppTask::Init()
+{
+	/* Initialize Matter stack */
+	ReturnErrorOnFailure(Nordic::Matter::InitServer(MatterEventHandler));
```

Comment:
nit: I wonder if it might be confusing for people to realize that `InitServer` doesn't actually initialize the Matter `Server` object. I am not sure if we can come up with a better name though (sth like `InitPlatform` perhaps), so you can leave it as it is.

Reply:
Maybe `PrepareServer` would work?

Reply:
Yes, I think it's better.